src/phoenix/server/api/routers/v1/experiments.py

```diff
@@ -252,3 +252,62 @@ async def get_experiment(request: Request, experiment_id: str) -> GetExperimentR
             updated_at=experiment.updated_at,
         )
     )
+
+
+class ListExperimentsResponseBody(ResponseBody[List[Experiment]]):
+    pass
+
+
+@router.get(
+    "/datasets/{dataset_id}/experiments",
+    operation_id="listExperiments",
+    summary="List experiments by dataset",
+    response_description="Experiments retrieved successfully",
+)
+async def list_experiments(
+    request: Request,
+    dataset_id: str,
+    project_name: Optional[str] = Query(
+        default=None, description="The project name to get experiments from"
+    ),
+) -> ListExperimentsResponseBody:
+    dataset_gid = GlobalID.from_id(dataset_id)
+    try:
+        dataset_rowid = from_global_id_with_expected_type(dataset_gid, "Dataset")
+    except ValueError:
+        raise HTTPException(
+            detail=f"Dataset with ID {dataset_gid} does not exist",
+            status_code=HTTP_404_NOT_FOUND,
+        )
+    async with request.app.state.db() as session:
+        query = (
+            select(models.Experiment)
+            .where(models.Experiment.dataset_id == dataset_rowid)
+            .order_by(models.Experiment.id.desc())
+        )
+        if project_name:
+            query = query.where(models.Experiment.project_name == project_name)
+
+        result = await session.execute(query)
+        experiments = result.scalars().all()
+
+        if not experiments:
+            return ListExperimentsResponseBody(data=[])
+
+        data = [
+            Experiment(
+                id=str(GlobalID("Experiment", str(experiment.id))),
+                dataset_id=str(GlobalID("Dataset", str(experiment.dataset_id))),
+                dataset_version_id=str(
+                    GlobalID("DatasetVersion", str(experiment.dataset_version_id))
+                ),
+                repetitions=experiment.repetitions,
+                metadata=experiment.metadata_,
+                project_name=experiment.project_name,
```

Comment:
I would hide this for now

Reply:
We currently need to return the project name with this payload, it's how we're currently returning experiments via the API and fails model validation if we don't, I can pass an empty string if we want to obfuscate this for the time being?